ipa-core/src/hpke/registry.rs

```diff
@@ -102,13 +146,22 @@ impl KeyRegistry<KeyPair> {
             keys: keys.into_boxed_slice(),
         }
     }
+}
 
+impl PrivateKeyRegistry for KeyRegistry<KeyPair> {
```

Comment:
with this change, do we have a use case to keep `KeyRegistry<KeyPair>` outside of test module ?

Reply:
Yes, it's still used in `ipa-core/src/cli/keygen.rs`.